src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      class="card__picture"
+      alt="IMacPhoto"
+      >
+      <h2 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__review-container">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <div class="card__reviews">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__price-container">
+        <div class="card__pricelabel">Price:</div>
+        <div class="card__price">$2,199</div>
+      </div>
+      <a
+      href="#"
+      class="card__link"
+      data-qa="hover"
+      >
+      Buy
+      </a>
+    </div>
```

Comment:
please pay attention to formatting your code:
```suggestion
    <div class="card" data-qa="card">
      <img
        src="images/imac.jpeg"
        class="card__picture"
        alt="IMacPhoto"
      >
      
      <h2 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
      
      <div class="card__code">Product code: 195434</div>
      
      <div class="card__review-container">
        <div class="stars stars--4">
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
        </div>
        
        <div class="card__reviews">
          Reviews: 5
        </div>
      </div>
      
      <div class="card__price-container">
        <div class="card__pricelabel">Price:</div>
        <div class="card__price">$2,199</div>
      </div>
      
      <a
        href="#"
        class="card__link"
        data-qa="hover"
      >
        Buy
      </a>
    </div>
```